File: convert.js
const docxConverter = require('docx-pdf');

const path = require('path');
const fs = require('fs')


const glob = require("glob");
let fileArr;
const Output_src = path.join(__dirname, "/")
console.log(Output_src);
const src = path.join(__dirname, "/test")

var getDirectories = function (src, callback) {
  glob(src + '/**/*', callback);
};
getDirectories('test', function (err, res) {
  if (err) {
    console.log('Error', err);
  } else {
        console.log(res)
        res.forEach((file) => {
            if (file.includes("docx")){    
              let filenameArr = file.split('/');
              let filename = `${filenameArr[filenameArr.length-2]}-${filenameArr.pop()}`
              const dst =  `Output/${filename}.pdf`;
              docxConverter(file, dst, (err, result) => {
                  if (err) console.log(err);
                  else console.log(result); // writes to file for us
              });
            } 
        })
    }
});

// 

File: convert.py
import sys
import os, stat
from docx2pdf import convert

NO_OTHER_WRITING = ''

path = os.path.dirname(__file__)
inputPath = os.path.join(path, 'test')
print("inputpath", inputPath)

for file in os.listdir(inputPath):
	filename = os.fsdecode(file)
	print(filename);
	if filename.endswith('.docx'):
		src = os.path.join(inputPath, filename)
		print(src);
		os.chmod(src, int('0777'))
		outputPath = os.path.join(path, f'{filename}.pdf')
		print(outputPath)
		convert(src,outputPath)